specs/binding-mqtt.spec/src/main/scripts/io/aklivity/zilla/specs/binding/mqtt/config/server.route.non.default.yaml

```diff
@@ -30,4 +30,3 @@ bindings:
           - subscribe:
               - topic: reply
         exit: app1
```

Comment:
Typically we use `net0` and `app0` for all config scenarios, let's do the same here if possible.